2.process-data/scripts/nbconverted/0.random-test-split.py

```diff
@@ -2,7 +2,7 @@
 # coding: utf-8
 
 # # Randomly Split Test Partition from Training
-# 
+#
```

Comment:
These files should not be updated - I assume they are updated with `black`?

The reason this file should not be updated is because it is [autogenerated](https://github.com/cytodata/single-cell-classifier/blob/master/2.process-data/process-data-pipeline.sh#L8-L10).

Because it is autogenerated, if its changed here, it will appear as a changed file the next time someone else runs the full `2.process-data` pipeline script. Does this make sense?

I added #29 to make sure this is specified in the `CONTRIBUTING.md` file.

Reply:
Good point. I ran indeed black on the whole repo